project/project.go

```diff
@@ -317,16 +318,25 @@ func (p *Project) startService(wrappers map[string]*serviceWrapper, history []st
 	return nil
 }
 
-func (p *Project) traverse(selected map[string]bool, wrappers map[string]*serviceWrapper, action wrapperAction, cycleAction serviceAction) error {
+func (p *Project) traverse(start bool, selected map[string]bool, wrappers map[string]*serviceWrapper, action wrapperAction, cycleAction serviceAction) error {
```

Comment:
@cpuguy83 I agree somewhat.  I don't want to change this though right now.  The problem is that project.go really needs to be completely refactored.  It is impossible to read and not flexible enough.  Because of this I rather just address this immediate bug and then work on a refactor.